src/widget/wtracktableview.cpp

```diff
@@ -869,6 +892,26 @@ QList<TrackId> WTrackTableView::getSelectedTrackIds() const {
     return trackIds;
 }
 
+TrackId WTrackTableView::getCurrentTrackId() const {
+    QItemSelectionModel* pSelectionModel = selectionModel();
+    VERIFY_OR_DEBUG_ASSERT(pSelectionModel != nullptr) {
+        qWarning() << "No selected tracks available";
+        return TrackId();
```

Comment:
You can simply use `return {}` in these cases to avoid repeating the type name over and over again. The actual type is visible in the signature above.

Reply:
Thanks!